src/cli/main.c

```diff
@@ -66,6 +67,9 @@ main(int argc, char *argv[])
 			del_network = 1;
 			network_name = optarg;
 			break;
+		case 'v':
+			version = NVAGENT_VERSION;
```

Comment:
mmm I wonder here if it would be less confusing to use `version` as a flag, et set it to 1 (version = 1),
then check if (version) and then print NVAGENT_VERSION